src/backend/joanie/core/management/commands/generate_certificates.py

```diff
@@ -85,12 +85,12 @@ def handle(self, *args, **options):
 
         filters = {}
         if order_uids:
-            filters.update({"uid__in": order_uids})
+            filters.update({"id__in": order_uids})
         else:
             if course_codes:
                 filters.update({"course__code__in": course_codes})
             if product_uids:
-                filters.update({"product__uid__in": product_uids})
+                filters.update({"product__id__in": product_uids})
```

Comment:
```suggestion
                filters.update({"product__id__in": product_ids})
```

Reply:
ditto the name of the var is product_uids, i've left it named this way as it is uuid